src/prefect/flow_runs.py

```diff
@@ -0,0 +1,37 @@
+from uuid import UUID
+
+import anyio
+
+from prefect.client.orchestration import PrefectClient
+from prefect.client.schemas import FlowRun
+from prefect.exceptions import FlowWaitTimeout
+
+
+async def wait_for_flow_run(
+    client: PrefectClient,
+    flow_run_id: UUID,
+    timeout: int = 10800,
```

Comment:
It'd be nice to type this a `Optional` since passing `None` will disable the timeout.
```suggestion
    timeout: Optional[int] = 10800,
```

Reply:
addressed